runtime/j9vm/javanextvmi.cpp

```diff
@@ -351,6 +349,8 @@ virtualThreadMountBegin(JNIEnv *env, jobject thread)
 		enterVThreadTransitionCritical(currentThread, thread);
 		threadObj = J9_JNI_UNWRAP_REFERENCE(thread);
 	}
+
+	VM_VMHelpers::virtualThreadHideFrames(currentThread, JNI_TRUE);
```

Comment:
Moved it after the suspend loop; we should wait for the virtual thread to be resumed before invoking this function.